edgelet/iotedge/src/main.rs

```diff
@@ -237,6 +240,26 @@ fn run() -> Result<(), Error> {
                 ),
         )
         .subcommand(SubCommand::with_name("version").about("Show the version information"))
+        .subcommand(
+            SubCommand::with_name("support-bundle")
+                .about("Bundles troubleshooting information")
+                .arg(
+                    Arg::with_name("location")
```

Comment:
Shouldn't it be a filename rather than a directory? The command only produces one archive file, right?